io/private/EptSupport.hpp

```diff
@@ -53,9 +55,75 @@
 #include <pdal/util/Bounds.hpp>
 #include <pdal/util/Utils.hpp>
 
+#include <arbiter/arbiter.hpp>
```

Comment:
Same comment on throwing things from this class.  If it were me, I'd throw an ept_error or some such and catch/rethrow in the caller.  As an alternative, you can save the stage somewhere and then call throwError().

Reply:
Error handling for exceptions in EptSupport utilities cleaned up in 4cc05e2.  The EPT reader/addon-writer now catch and re-throw the errors with `throwError`, and an `ept_error` descended from `pdal_error` was created for these exceptions (although this exception type is not currently treated specially).